NuRadioReco/modules/RNO_G/stationLDAVariables.py

```diff
@@ -61,16 +62,16 @@ def run(self, event, station, detector, channel_id=0):
             reference channel for the coherent sum
 
         """
+        channelSignalReconstructor=NuRadioReco.modules.channelSignalReconstructor.channelSignalReconstructor()
+        channelSignalReconstructor.begin()
+        channelSignalReconstructor.run(event, station, detector)
```

Comment:
This is a bad idea. The user has to make sure to include the module in their sequence